webapp-django/crashstats/supersearch/jinja2/supersearch/macros/date_filters.html

```diff
@@ -18,6 +23,7 @@
     <a href="#" data-range="2w">14 days</a> -
     <a href="#" data-range="1M">1 month</a> -
     <a href="#" data-range="3M">3 months</a> -
-    <a href="#" data-range="6M">6 months</a>
+    <a href="#" data-range="6M">6 months</a> |
+    <a href="#" data-range="dayends">day ends</a>
```

Comment:
The "dayends" thing sets the "from" time to 12:00am and the "to" time to 11:59pm. It's kind of nice.